Sanity/main.fmf

```diff
@@ -4,6 +4,7 @@ contact: Martin Zelený <mzeleny@redhat.com>
 component:
   - tang
 test: ./runtest.sh
+framework: beakerlib
```

Comment:
In the common test repository, we have it in the root node and let the inheritance to work for us.
https://github.com/RedHat-SP-Security/tests/blob/master/main.fmf

Reply:
Good idea, I'll adjust it